cibuild.cmd

```diff
@@ -107,3 +100,14 @@ exit /b 0
 @echo            with /release. May not be combined with /test32 or /test64.
 @echo.
 @goto :eof
+
+:BuildFailed
+echo Build failed with ERRORLEVEL %ERRORLEVEL%
+call :TerminateCompilerServer
+exit /b 1
+
+:TerminateCompilerServer
+@REM Kill any instances VBCSCompiler.exe to release locked files, ignoring stderr if process is not open
+@REM This prevents future CI runs from failing hile trying to delete those files.
+
+taskkill /F /IM vbcscompiler.exe 2> nul
```

Comment:
@amcasey hmm, that's interesting.  That would be a problem.  Reason 450 that we shouldn't be running jobs in parallel.